Classes/Service/ExtensionService.php

```diff
@@ -94,4 +95,16 @@ public function isStoragePathConfigured(): bool
     {
         return !Environment::isComposerMode() || count($this->resolveStoragePaths()) > 0;
     }
+
+    /**
+     * Returns, if typo3/cms-composer-installers v4 is used by checking, if ext:extension_builder is
+     * installed in /vendor/friendsoftypo3/extension-builder/
+     *
+     * @return bool
+     */
+    public function isV4ComposerInstaller(): bool
+    {
+        $extensionPath = GeneralUtility::getFileAbsFileName('EXT:extension_builder/');
+        return strpos($extensionPath, '/vendor/friendsoftypo3/extension-builder/') !== false;
```

Comment:
str_starts_with should be used

Reply:
Changed this to `str_contains`, since actually it is checked, if `/vendor/friendsoftypo3/extension-builder/` is part of the extensions install path